connectors/slack/element-templates/slack-connector.json

```diff
@@ -161,6 +161,28 @@
         "equals": "conversations.create"
       }
     },
+    {
+      "id": "message.type",
+      "label": "Message type",
+      "group": "message",
+      "type": "Dropdown",
+      "value": "plain_text",
```

Comment:
Please use `camelCase` as we usually do.

Reply:
Saying this as we should migrate this one to the element template generator and we should strive for keeping existing patterns for enums.

Reply:
Done

Reply:
thx!